com.unity.template.renderstreaming-hd/Assets/Scripts/RenderStreaming.cs

```diff
@@ -24,10 +24,13 @@ public class ButtonClickElement
 
     public class RenderStreaming : MonoBehaviour
     {
-#pragma warning disable 0649
-        [SerializeField, Tooltip("Address for signaling server")]
+        #pragma warning disable 0649
+        [SerializeField, Tooltip("Signaling server url")]
         private string urlSignaling = "http://localhost";
 
+        [SerializeField, Tooltip("Type of signaling server")]
+        private string signalingType = typeof(HttpSignaling).FullName;
```

Comment:
I guess it should be enum parameter. `InspectorNameAttribute` is useful for this case.
https://docs.unity3d.com/2020.1/Documentation/ScriptReference/InspectorNameAttribute.html

Reply:
I proposed this concept to allow each developper to create it's own signaling server connection (ISignaling) and to automatically add this implementation in the drop down list. The smart side of this is that it isn't necessary to know the user-dependent implementation from RenderStreaming code. An enum may be too restrictive because we have to add each class in RenderStreaming code.